src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -1,18 +1,17 @@
 package uk.gov.nationalarchives.tdr.api.service
 
-import java.sql.Timestamp
-import java.util.UUID
-
 import uk.gov.nationalarchives.Tables.{FileRow, FilemetadataRow}
 import uk.gov.nationalarchives.tdr.api.db.repository._
 import uk.gov.nationalarchives.tdr.api.graphql.fields.FileFields.{AddFileAndMetadataInput, ClientSideMetadataInput, FileMatches}
 import uk.gov.nationalarchives.tdr.api.model.file.NodeType
```

Comment:
Don't think this import is needed anymore: `import uk.gov.nationalarchives.tdr.api.model.file.NodeType`

Reply:
I've deleted it, cheers.